ckanext/dalrrd_emc_dcpr/templates/dcpr/snippets/request_form.html

```diff
@@ -175,6 +175,7 @@ <h2>DATA INFORMATION</h2>
         </div>
 
     </fieldset>
+    {%  if is_nsif_member or is_sysadmin %}
```

Comment:
@Fanevanjanahary where is_nsif_member defined?, on other places it's defined as follows (along with csi):
{% set is_nsif_member = h.emc_user_is_org_member('nsif', g.userobj) %}
{% set is_csi_member = h.emc_user_is_org_member('csi', g.userobj) %}